Analytics/Classes/Internal/SEGHTTPClient.h

```diff
@@ -15,6 +15,8 @@ NS_ASSUME_NONNULL_BEGIN
 @interface SEGHTTPClient : NSObject
 
 @property (nonatomic, strong) SEGRequestFactory requestFactory;
+@property (nonatomic, readonly) NSMutableDictionary<NSString *, NSURLSession *> *sessionsByWriteKey;
```

Comment:
There's only ever one writeKey (the client must be a singleton), so this will only have one entry? In which case do we need a map>